tools/licenses/lib/filesystem.dart

```diff
@@ -241,6 +242,7 @@ FileType identifyFile(String name, Reader reader) {
     case '.icc': return FileType.binary; // Color profile
     case '.swp': return FileType.binary; // Vim swap file
     case '.bfbs': return FileType.binary; // Flatbuffers Binary Schema
+    case '.rom': return FileType.binary;
```

Comment:
.rom-files that are part of fuchsia/sdk/linux/tools should be already skipped by https://github.com/flutter/engine/blob/main/tools/licenses/lib/paths.dart#L40.
Were you able to confirm where other .rom-files are coming?

Reply:
Yea, it's in the linked issue: `../../../fuchsia/sdk/core.tar.gz!core.tar!/tools/arm64/aemu_internal/lib/pc-bios/efi-e1000.rom`.

Reply:
Were you able to confirm where core.tar.gz came from? I don't think it's part of fuchsia sdk.

Reply:
Nope, I didn't look deeply into it since this is a blocker and it's there somehow, and in no case do we want to do license checks on rom files.  I assume its presence is justified somewhere else since the engine code isn't doing it.

Reply:
I would be wary of this being another symptom of some issue with newly introduce license builder. Similar to how 'gclient' was missing, for example.

Reply:
I rebased srujans PR to make sure this is still an issue after godofredos change to the bot.  There still may be some oddity with the bot, but we can unblock the rolls while its investigated.

Reply:
(I just verified that srujans PR still needs this fix)

Reply:
actually, every PR needs this

Reply:
if we're not expecting the core.tar.gz file we should figure out where it is coming from.

Reply:
(especially a .tar.gz file, which is probably adding more time to the already slow license script process)

Reply:
I filed an issue: https://github.com/flutter/flutter/issues/126475